.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Report a defect in the software
+
+---
+
+<!--
+Thank you for your bug report.
+
+NOTE:
+
+    If you're asking about how to use OpenSSL, this isn't the right 
+    forum.  Please see our
+    [User Support resources](https://github.com/openssl/openssl/blob/master/.github/SUPPORT.md)
+
+PLEASE remember to tell us in what OpenSSL version you found the issue.
```

Comment:
Side note: Is the caps lock on "PLEASE" intentional?

Reply:
Uhmmm, it was.  Too much?

Reply:
Don't know! 

For me it was just unexpected: I thought "we are either VERY polite or it was a typo!"

Reply:
Fixed

Reply:
So we are not THAT polite! :P

Reply:
Well, "PLEASE" signals a raised voice, which is reasonable to see as a sign of frustration.  So on the politeness scale, "Please" ends up being *more* polite :wink: